freqtrade/commands/cli_options.py

```diff
@@ -202,6 +202,11 @@ def __init__(self, *args, **kwargs):
         help='Specify additional lookup path for Hyperopt and Hyperopt Loss functions.',
         metavar='PATH',
     ),
+    "backtest_path": Arg(
```

Comment:
I'm ... not quite sure what you have in mind with this (doesn't seem to be used beyond this point) - but maybe `exportfilename` covers this ... ?

Reply:
I had built a small text "parser" locally for the backtesting logfile and assumed I would build the filter on the logfile. That was before I realized that a yaml file was available for processing. I will think about this on my next codingrun and possible will remove this.